context/app/static/js/components/Detail/Citation/Citation.jsx

```diff
@@ -20,7 +21,13 @@ function Citation({ contributors, title, timestamp, url }) {
 
   return (
     <SectionContainer id="citation">
-      <SectionHeader>Citation</SectionHeader>
+      <SectionHeader>
```

Comment:
For a quick fix, adding `display: flex; align-items:center;` to a styled section header should fix the alignment issue.

Reply:
Thanks!